float32[3] tag1
float32[3] tag2
float32[3] tag3
float32[3] tag4
float32[3] tag5
float32[3] tag6
float32[3] tag7
float32[3] tag8
float32[3] tag9
float32[3] tag10
float32[3] tag11
float32[3] tag12
float32[3] tag13
float32[3] tag14
---
float32 score
